lib/api/core/storage/clientAdapter.js

```diff
@@ -148,6 +150,17 @@ class ClientAdapter {
     return response;
   }
 
+  // @todo test unit
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)